scripts/evaluate_bidaf.sh

```diff
@@ -28,24 +28,27 @@ bidaf_input=${input_file_prefix}_with_paras_${run_name}.jsonl
 if [ ! -f ${input_file_with_hits} ]; then
 	python arc_solvers/processing/add_retrieved_text.py \
 		${input_file} \
-		${input_file_with_hits}
+		${input_file_with_hits}.$$
+	mv ${input_file_with_hits}.$$ ${input_file_with_hits}
 fi
 
 # Merge hits for each question
 if [ ! -f ${bidaf_input} ]; then
 	python arc_solvers/processing/convert_to_para_comprehension.py \
-	${input_file_with_hits} \
-	${input_file} \
-	${bidaf_input}
+    ${input_file_with_hits} \
```

Comment:
Nit pick: Can you increase the indent ? I like having the lines in a multi-line command indented at least as much as, preferably more than, the first line of the command. Makes it clearer when a new command is starting.

Reply:
Looks like there were some tabs vs. spaces issues going on. I fixed them.